code/modules/mob/living/simple_animal/animals/cat.dm

```diff
@@ -156,6 +156,7 @@
 /mob/living/simple_animal/cat/fluff/Runtime
 	name = "Runtime"
 	desc = "Her fur has the look and feel of velvet, and her tail quivers occasionally."
+	tt_desc = "Felis medicalis"

```

Comment:
every single one of these names bothers me personally and i still think anything you can directly see outside of examines should be only in-character